pytorch_lightning/accelerators/ddp2_accelerator.py

```diff
@@ -66,6 +66,9 @@ def validation_step(self, args):
     def test_step(self, args):
         return self._step(args)
 
+    def forward(self, args):
+        return self._step(args)
+
```

Comment:
```suggestion
    def forward(self, args):
        return self._step(args)

```
can we move this to parent class?

Reply:
<img width="333" alt="Screenshot 2021-01-20 at 20 01 05" src="https://user-images.githubusercontent.com/12861981/105228261-4784c580-5b5a-11eb-9ae4-cf8cb36fcd5b.png">

Reply:
I kept it this way for consistency. If we decide to do so, let's move them all to base accelerator.
But this will be another PR.